src/transformStateWithClones.js

```diff
@@ -6,8 +6,35 @@
  *
  * @return {Object[]}
  */
+
 function transformStateWithClones(state, actions) {
-  // write code here
+  let stateCopy = { ...state };
+  const states = [];
```

Comment:
```suggestion
  const transformState = [];
```
 pay attention on naming